no-computer.md

```diff
@@ -58,12 +98,24 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think it would be split about 50/50 between language aspects and systems aspects. To make
+a system that makes noise based on button presses can be tricky so less time can be spent
+on the language aspects than desired. Either way this will require a lot of thought to
+allow for proper fluency and chaining of notes in a way that is easy for the users.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is quite an ambitious project. It will require engineering design and good understanding
+of what allows for easier music creation. 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a good idea for a project since it is very useful and includes lots of imaginative
+language design aspects. This could be a bad idea because it requires too much engineering aspects
+and could not be the optimal way to interact with music. 
+
+On a side note, I deliberately did not mention what Non-computer my language is and I
```

Comment:
I was a bit confused as well, although piano (or some keyboard-based instrument) is still my guess.